### ISSUE TITLE
AttributeError: 'FreeTypeFont' object has no attribute 'getsize'

### ISSUE THREAD

**Issue by Abdur-rahmaanJ** [cc-d/flask-simple-captcha#14]
Getting this error
```
    captcha = CAPTCHA.create()
              ^^^^^^^^^^^^^^^^
/flask_simple_captcha/__init__.py", line 79, in create
    text_size = fnt.getsize(text)
                ^^^^^^^^^^^^^^^^^
AttributeError: 'FreeTypeFont' object has no attribute 'getsize'
```

**Comment by cc-d** [comment]
this project has been kinda abandoned for a while, but since i've been super active recently and saw your issue i fixed it. the issue was pillow dumb me 5 years ago didn't put an upper limit on pillow package version, 10.0 seems to be broke i changed the range to 6-9 and this should now be fixed in `1.2.0`

**Comment by Abdur-rahmaanJ** [comment]
Hey @cc-d Thanks a lot. Btw for [my project](https://github.com/shopyo/shopyo) Pillow was a real pain. I mistakenly included it in the deps for years and it pains me whenever i have to install older version.

**Comment by TruncatedDinosour** [comment]
> this project has been kinda abandoned for a while, but since i've been super active recently and saw your issue i fixed it. the issue was pillow dumb me 5 years ago didn't put an upper limit on pillow package version, 10.0 seems to be broke i changed the range to 6-9 and this should now be fixed in `1.2.0`

it isnt fixed btw
